database.hh

```diff
@@ -1233,7 +1234,7 @@ private:
     lw_shared_ptr<db_stats> _stats;
     std::unique_ptr<cell_locker_stats> _cl_stats;
 
-    std::unique_ptr<db::config> _cfg;
+    const db::config* _cfg;
```

Comment:
Can this be a reference? It is always passed in the constructor and is never changed to point to anything else.

Reply:
I made it a pointer to avoid changing all the call sites. It would be better as a reference and I can follow up with a patch to fix that up, but I don't think it should be a part of this patchset.